### PR TITLE
EL7 Fixed

### PR DESCRIPTION
**Pull request by tdegeorge**

### CODE REVIEW

File: source/05-EL/07.ptx [hunk 1]
```diff
@@ -59,12 +59,12 @@
...
-              <li><m>70</m></li>
-              <li><m>20</m></li>
-              <li><m>0</m></li></ol>
+              <li><m>90</m><degree/><m>C</m></li>
+              <li><m>70</m><degree/><m>C</m></li>
+              <li><m>20</m><degree/><m>C</m></li>
+              <li><m>0</m><degree/><m>C</m></li></ol>
           </p>
         </statement>
         <answer>
```
Comment (siwelwerd) [inline source/05-EL/07.ptx:70]:
```suggestion
<hint><p>Think about what happens to the function as <m>x\to \infty</m>.</p></hint>
        <answer>
```

File: source/05-EL/07.ptx [hunk 2]
```diff
@@ -59,12 +59,12 @@
       <task>
         <statement>
           <p>
-            According to the function given, if Mr. Abacus leaves his coffee on his desk all day, what will the coffee eventually cool to?
+            According to the function given, if Mr. Abacus leaves his coffee on his desk all day, what will the coffee eventually cool to? (Hint: Think about what happens to the function as <m>x\to \infty</m>.)
```
Comment (siwelwerd) [inline source/05-EL/07.ptx:62]:
```suggestion
            According to the function given, if Mr. Abacus leaves his coffee on his desk all day, what will the coffee eventually cool to?
```

**Comment by siwelwerd** [comment]
@tdegeorge See above suggestion to use the `<hint>` tag.

File: source/05-EL/07.ptx (hunk 1 above)
Comment (tdegeorge) [reply source/05-EL/07.ptx:70]: I think that looks better. I've made the changes.

**Comment by tdegeorge** [comment]
Ok...made the change and tried to submit it through. I *believe* it worked?

**Review by siwelwerd — APPROVED** [review]

**Comment by siwelwerd** [comment]
> I believe it worked?

Yep!

**Pull request merged**